### DISCUSSION TITLE
Get a `RwSignal` from `create_slice`

### DISCUSSION THREAD

**Discussion by MForster** [leptos-rs/leptos#2153] [Q&A] [answered]
Is there a way to create a `RwSignal` for a slice? Or can I somehow else merge the `Signal`/`SignalSetter` pair returned by `create_slice` into a single signal that allows all operations?

**Comment by MForster** [comment]
Oh, I guess that's not possible, because there is no way to implement `update`?

**Reply by gbj** [reply, marked as answer]
That's exactly right! Also RwSignal just means a very particular thing (this is an atomic unit of state). 

I've been working on a Store interface for the next version that would allow RW reactive access to struct fields without all the slicing business, so... good things coming in the future but not yet.
